### PR TITLE
implement minimal frontend

### PR DESCRIPTION
**Pull request by akhileshh**
* attribute metadata endpoint
* attribute values for l2ids endpoint

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,4 +6,5 @@ RUN mkdir -p /home/nginx/.cloudvolume/secrets && chown -R nginx /home/nginx && u
 COPY . /app
 RUN pip install pip==20.0.1 \
     && pip install --no-cache-dir --upgrade -r requirements.txt \
-    && pip install --upgrade git+https://github.com/seung-lab/PyChunkedGraph.git@master
\ No newline at end of file
+    && pip install --upgrade git+https://github.com/seung-lab/PyChunkedGraph.git@master \
```
Comment (sdorkenw) [inline Dockerfile:9]: Why is the PyChunkedGraph required?

File: pcgl2cache/app/v1/routes.py [hunk 2]
```diff
@@ -57,8 +44,19 @@ def unhandled_exception(e):
...
 def api_exception(e):
     return common.api_exception(e)
 
 
+@bp.route("/attribute_metadata", methods=["GET"])
+def attr_metadata():
+    return jsonify_with_kwargs(common.handle_attr_metadata())
+
+
+@bp.route("/table/<table_id>/attributes", methods=["POST"])
```
Comment (sdorkenw) [inline pcgl2cache/app/v1/routes.py:57]: auth is currently missing. Can add that later

**Review by sdorkenw — APPROVED** [review]
Looks good!

File: Dockerfile (hunk 1 above)
Comment (akhileshh) [reply Dockerfile:9]: for ingest, but we could maintain two docker files, it is not required to run frontend

**Pull request merged**